interactive_ai/workflows/otx_domain/trainer/otx_v2/Dockerfile.xpu

```diff
@@ -0,0 +1,53 @@
+FROM python:3.10-slim-bookworm AS base
+
+ENV UV_COMPILE_BYTECODE=1
+ENV UV_LINK_MODE=copy
+ENV UV_PYTHON_DOWNLOADS=0
+
+# Install runtime dependencies and intel graphics libs
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    libgl1=1.6.* \
+    libglib2.0-0=2.74.* \
+    software-properties-common \
+    gpg \
+    wget \
```

Comment:
Thanks, Leo!

Some utilities like `gpg`, `wget`, and others are only required during the build stage. Also, building in a runtime image results in a high number of layers.

Could you please separate the build and runtime stages to produce a clean, minimal runtime image?

Reply:
> Could you please separate the build and runtime stages to produce a clean, minimal runtime image?

This was actually my first attempt (commit 22731638). While the separation saves ~40mb from the final image size (~8GB), building it requires significant more disk space: in practice, the single-stage Dockerfile fits in the standard GHA runner (14GB disk space), while the multi-stage one does not. Given the difficulties to get access to a more powerful runner, imho the compatibility with the standard runner outweighs the minimal difference in image size. I can work anyway to optimize it by uninstalling gpg and the other build packages at the end.

Reply:
Removed gpg, wget and other unnecessary files in 30a780fa